ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activity-answer/activityAnswer.component.ts

```diff
@@ -8,61 +8,71 @@ import { BlockVisibility } from '../../../../models/activity/blockVisibility';
+                                         [readonly]="readonly"
+                                         (valueChanged)="onChange($event)">
+            </ddp-activity-numeric-answer>
+            <ddp-activity-picklist-answer *ngIf="isPicklistQuestion(block)"
+                                          [class]="'picklist-answer-' + block.stableId"
+                                          [block]="block"
+                                          [readonly]="readonly"
+                                          (valueChanged)="onChange($event)">
+            </ddp-activity-picklist-answer>
+            <ddp-activity-date-answer *ngIf="isDateQuestion(block)"
+                                      [class]="'date-answer-' + block.stableId"
+                                      [block]="block"
+                                      [readonly]="readonly"
+                                      [validationRequested]="validationRequested"
+                                      (valueChanged)="onChange($event)">
+            </ddp-activity-date-answer>
+            <ddp-activity-composite-answer *ngIf="isCompositeQuestion(block)"
+                                           [class]="'composite-answer-' + block.stableId"
+                                           [block]="block"
+                                           [readonly]="readonly"
+                                           [validationRequested]="validationRequested"
+                                           (valueChanged)="onChange($event)">
+            </ddp-activity-composite-answer>
+            <ddp-activity-agreement-answer *ngIf="isAgreementQuestion(block)"
+                                           [block]="block"
+                                           [readonly]="readonly"
+                                           (valueChanged)="onChange($event)">
+            </ddp-activity-agreement-answer>
+            <ddp-activity-file-answer *ngIf="isFileQuestion(block)"
+                                      [class]="'file-answer-' + block.stableId"
+                                      [block]="block"
+                                      [readonly]="readonly"
+                                      [studyGuid]="studyGuid"
+                                      [activityGuid]="activityGuid"
+                                      (valueChanged)="onChange($event)">
+            </ddp-activity-file-answer>
+            <span *ngIf="block.additionalInfoFooter"
+                  [innerHTML]="block.additionalInfoFooter"
+                  class="ddp-activity-answer__info-footer">
       </span>
-    </ng-container>`,
+        </ng-container>`,
     styleUrls: ['activityAnswer.component.scss']
 })
 
 export class ActivityAnswerComponent {
     @Input() block: AbstractActivityQuestionBlock;
     @Input() readonly: boolean;
     @Input() validationRequested: boolean;
+    @Input() studyGuid: string;
+    @Input() activityGuid: string;
```

Comment:
Added `studyGuid` and `activityGuid` as to pass them to `activityFileAnswer.component.ts`